app/controllers/candidate_interface/gcse/english/grade_controller.rb

```diff
@@ -0,0 +1,38 @@
+module CandidateInterface
+  class Gcse::English::GradeController < CandidateInterfaceController
+    include Gcse::GradeControllerConcern
+
+    before_action :redirect_to_dashboard_if_submitted
+    before_action :set_subject
+
+    def edit
+      @application_qualification = details_form
```

Comment:
Worth revisiting this naming in the next bit of this work, especially if we split out the details form. Will probably be a `@grade_form` variable that we end up dealing with.